codegen/src/api/mod.rs

```diff
@@ -352,6 +356,72 @@ impl RuntimeGenerator {
     }
 }
 
+/// Most chains require a valid account nonce as part of the extrinsic, so the default behaviour of
+/// the client is to fetch the nonce for the current account (if it has not been specified).
+///
+/// The account nonce is commonly stored in the `System` pallet's `Account` storage item. This
```

Comment:
> I wonder if it's actually possible to build a chain that stores the nonce elsewhere?

It would involve a custom `frame_system` pallet. It's also possible that this will change across versions of `frame_system`, I know it has in the past

> Is "nonce" here the same as the "Account index"?

Yes indeed it is used interchangeably. Personally I find index slightly misleading, and the field is called nonce, but I can change the assoc type name here for consistency?